### DISCUSSION TITLE
Podman Desktop v1.2.0 is now out!

### DISCUSSION THREAD

**Discussion by cdrage** [podman-desktop/podman-desktop#3226] [Announcements]
A new release of Podman Desktop is out: [v1.2.0](https://github.com/containers/podman-desktop/milestone/21?closed=1)

Download it from the [download section](https://podman-desktop.io/downloads) of the [Podman Desktop website](https://podman-desktop.io/)

Release Notes are available at https://podman-desktop.io/blog/podman-desktop-release-1.2

Full Changelog is available at https://github.com/containers/podman-desktop/releases/tag/v1.2.0

**Comment by afbjorklund** [comment]
Flathub seems to lag a bit, https://flathub.org/apps/io.podman_desktop.PodmanDesktop still has 1.1.0

EDIT: Looks like the build (PR) is failing https://github.com/flathub/io.podman_desktop.PodmanDesktop

**Reply by cdrage** [reply]
Hi @afbjorklund yup, that's correct. We're still investigating why the build is failing: https://github.com/flathub/io.podman_desktop.PodmanDesktop/pull/25

For now, you can still grab the flatpak here: https://github.com/containers/podman-desktop/releases/tag/v1.2.0 and install it manually.
